nmmo_env.py

```diff
@@ -92,38 +94,69 @@ def __init__(self, args: Namespace):
 
 
 class Postprocessor(pufferlib.emulation.Postprocessor):
-  # def __init__(self, env, teams, team_id):
-  #   super().__init__(env, teams, team_id)
-  # self._feature_extractor = FeatureExtractor(teams, team_id, env.config)
+  def __init__(self, env, teams, team_id):
+    super().__init__(env, teams, team_id)
 
-  # def reset(self, team_obs):
-  #   super().reset(team_obs)
-  # self._feature_extractor.reset(team_obs)
+    # stat vars
+    self._cod_attacked = 0
```

Comment:
how about moving these into _reset_episode_stats() so that we only have one place to modify

Reply:
added _reset_episode_stats()